src/frontend/Sider/Journalføring/Standard/DokumentPanel.tsx

```diff
@@ -0,0 +1,165 @@
+
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+
+import { ExternalLinkIcon } from '@navikt/aksel-icons';
+import { Button, ExpansionCard } from '@navikt/ds-react';
+import { FamilieReactSelect, ISelectOption } from '@navikt/familie-form-elements';
+
+import { DokumentPanelHeader } from './DokumentPanelHeader';
+import { JournalføringState } from '../../../hooks/useJournalføringState';
+import { DokumentInfo } from '../../../typer/dokument';
+import { åpneFilIEgenTab } from '../../../utils/utils';
+import {
+    dokumentTitlerMultiSelect,
+    mapLogiskeVedleggTilMultiselectValue,
+    mapDokumentTittelTilMultiselectValue,
+    mapMultiselectValueTilLogiskeVedlegg,
+} from '../Felles/utils';
+
+const ExpansionCardHeader = styled(ExpansionCard.Header)`
+    padding-bottom: 0.35rem;
+`;
+
+const ExpansionCardContent = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    padding-bottom: 1rem;
+`;
+
+const EksternLenkeKnapp = styled(Button)`
+    width: fit-content;
+`;
+
+const MultiSelect = styled(FamilieReactSelect)`
+    margin-bottom: -1rem;
+`;
+
+interface Props {
+    journalpostState: JournalføringState;
+    dokument: DokumentInfo;
+}
+
+const DokumentPanel: React.FC<Props> = ({ journalpostState, dokument }) => {
+    const {
+        dokumentTitler,
+        logiskeVedleggPåDokument,
+        settDokumentTitler,
+        settLogiskeVedleggPåDokument,
+        settValgtDokumentPanel,
+        valgtDokumentPanel,
+        journalpost,
+    } = journalpostState;
+
+    const endreDokumentNavn = (dokumentInfoId: string, nyttDokumentNavn: string) => {
+        settDokumentTitler((prevState: Record<string, string> | undefined) => ({
+            ...prevState,
+            [dokumentInfoId]: nyttDokumentNavn,
+        }));
+    };
+
+    const endreLogiskeVedlegg = (dokumentInfoId: string, nyeLogiskeVedlegg: string[]) => {
+        settLogiskeVedleggPåDokument((prevState) => ({
+            ...prevState,
+            [dokumentInfoId]: nyeLogiskeVedlegg.map((tittel) => {
+                return { logiskVedleggId: uuidv4(), tittel: tittel };
+            }),
+        }));
+    };
+
+    const dokumentPanelErValgt = valgtDokumentPanel === dokument.dokumentInfoId;
+    const dokumentTittel =
+        (dokumentTitler && dokumentTitler[dokument.dokumentInfoId]) ||
+        dokument.tittel ||
+        'Ukjent dokumenttittel';
+    const defaultTittelValue = dokumentTittel
+        ? mapDokumentTittelTilMultiselectValue(dokumentTittel)
+        : undefined;
+
+    const logiskeVedlegg = logiskeVedleggPåDokument
+        ? logiskeVedleggPåDokument[dokument.dokumentInfoId]
+        : undefined;
+    const defaultLogiskeVedleggValue = logiskeVedlegg
+        ? mapLogiskeVedleggTilMultiselectValue(logiskeVedlegg)
+        : undefined;
+
+    // TODO: Når aksel sin combobox er ute av beta, bytt ut MultiSelect med den og fjern @navikt/familie-form-elements fra package.json
```

Comment:
Comboboxen er allerede brukt i sett på vent

Reply:
Den var litt vanskelig å ta i bruk siden den fortsatt er i beta. Den fikser ikke default-verdier for logiske vedlegg som allerede finnes på dokumentet så godt, så jeg tenkte man like greit kan vente med å legge den inn i journalføringen til den er "ferdig"

Reply:
Noterte ned det å ta i bruk aksel sin Combobox som en oppfølgingsoppgave her: https://favro.com/organization/98c34fb974ce445eac854de0/4d617346d79341c7fbd9a40a?card=NAV-19955